README.md

```diff
@@ -29,12 +29,93 @@ by using the Velero CSI plugin during backup of the stateful application.
 - Have a stateful application running in a separate namespace. 
 
 - [Install](https://volsync.readthedocs.io/en/stable/installation/index.html) the VolSync controller.
+
 ```
 $ helm repo add backube https://backube.github.io/helm-charts/
 $ helm install -n openshift-adp volsync backube/volsync
 ```
 
+- Install the VolumeSnapshotMover CRDs `VolumeSnapshotBackup` and `VolumeSnapshotRestore` using: `oc create -f config/crd/bases/`
+
+  - Example data mover CRs:
+
+```
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotBackup
+metadata:
+  name: <vsb-name>
+spec:
+  volumeSnapshotContent:
+    name: <snapcontent-name>
+  protectedNamespace: <adp-namespace>
+
+```
+
+```
+apiVersion: datamover.oadp.openshift.io/v1alpha1
+kind: VolumeSnapshotRestore
+metadata:
+  name: <vsr-name>
+spec:
+  protectedNamespace: <protected-ns>
+  resticSecretRef: 
+    name: restic-secret
+  dataMoverBackupRef:
+    sourcePVCData: 
+      name: <source-pvc-name>
+      size: <source-pvc-size>
+    resticrepository: <your-restic-repo>
+```
+
+- Create a DPA similar to below:
+
+```
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: bucket-name
+          prefix: bucket-prefix
+        provider: aws
+  configuration:
+    restic:
+      enable: false
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+      featureFlags:
+        - EnableCSI
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-west-2
+        provider: aws
+  unsupportedOverrides:
+    csiPluginImageFqin: 'quay.io/spampatt/velero-plugin-for-csi:latest'
```

Comment:
should we add an override for velero as well? the one with restore priority changed